### PR TITLE
add delete site to porcelain client

### PR DESCRIPTION
**Pull request by reed-acst**
In reference to https://github.com/netlify/open-api/issues/98

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,4 +4,5 @@ vendor
 yarn.lock
 dist
 .vscode
-.gobincache
\ No newline at end of file
+.gobincache
+.idea/*
```
Comment (mraerino) [inline .gitignore]: please put those into your global gitignore file, not in the repo. I know we have `.vscode` in there, but this should not prove as an example.

Comment (reed-acst) [reply .gitignore]: ✔️

**Comment by reed-acst** [comment]
@mraerino Made the requested changes. Can I get another review?

**Comment by elliehic** [comment]
Can we get another review of Reed's change? @mraerino @ehmicky

**Comment by reed-acst** [comment]
@mraerino can I get another review?

**Comment by reed-acst** [comment]
@mraerino can I get another review?

**Pull request merged**